tests/smoketest/events-smartgateway.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: smartgateway.infra.watch/v2alpha1
```

Comment:
If we deploy from quickstart.sh, then there will already be smartgateways deployed (called "saf-default-notification-smartgateway" and "saf-default-telemetry-smartgateway". It might make more sense if the smoke test used those ones instead of creating new ones.

Reply:
If we are testing already deployed SGs, we don't need this here at all.

Reply:
That's right, yeah. Ultimately the very most useful thing would be if the smoketest would look at your ServiceAssurance objects and then test what it finds to help customers verify that the topology they've deployed is working correctly. That seems like a big leap from here, though, so just testing the default topology that comes out of the box seems like a good first step for this activity.

Reply:
By testing the default scenario we can add other architecture deployment types by just passing a verbose ServiceAssurance manifest with all `*_manifest` parameters.

If a customer has a topology they want or we identify other ones we can pass and save for functional testing.

Reply:
As a follow up and to reaffirm what Chris said, that is work for another time, but by using the built in SGs that the SAO creates on an SA instantiation, then we'll be in better shape for that to eventually become a possibility.

Reply:
So this means we want to delete the `metrics-smartgateway.yaml` and the `events-smartgateway.yaml` then correct? We should only pass in a new file that creates a `ServiceAssurance` instance with metrics and events enabled.

Reply:
yes, that's the next step/cleanup

Reply:
> We should only pass in a new file that creates a ServiceAssurance instance with metrics and events enabled.

quickstart.sh already does this, I don't think it needs to be reproduced here.

Definitely waiting to see events-smartgateway.yaml, metrics-smartgateway.yaml, and the two associated `oc create -f` commands associated with them gone before I'll approve. The results query have been adjusted to look for metrics from "saf-default-telemetry-smartgateway", so these other smartgateway definitions are just noise.

Reply:
@csibbitt yea that makes sense, I just meant that the SGs should be created via that method (wherever it may live).